specificatie/partner.yaml

```diff
@@ -9,7 +9,7 @@ components:
     AbstractPartner:
       type: object
       description: |
-        Gegevens over een gesloten huwelijk/geregistreerd partnerschap van de persoon.
+        Gegevens over een gesloten of ontbonden huwelijk/geregistreerd partnerschap van de persoon.
```

Comment:
Voorstel: Gegevens over het actuele of het laatst ontbonden huwelijk/geregistreerd partnerschap van de persoon.

(dan wordt duidelijker dat niet de hele partnerhistorie wordt geleverd)

Reply:
Aangepast